caddyhttp/proxy/proxy_test.go

```diff
@@ -247,8 +247,8 @@ func TestWebSocketReverseProxyNonHijackerPanic(t *testing.T) {
 func TestWebSocketReverseProxyServeHTTPHandler(t *testing.T) {
 	// No-op websocket backend simply allows the WS connection to be
 	// accepted then it will be immediately closed. Perfect for testing.
-	var connCount int32
```

Comment:
I just tested this, and indeed, you're right...

As for sleeping vs. time.After, it's okay either way, I think the first way you did it made sense too. But I'll merge this!